evadb/expression/expression_utils.py

```diff
@@ -296,3 +298,20 @@ def _has_simple_expressions(expr):
     ]
 
     return _has_simple_expressions(predicate) and contains_single_column(predicate)
+
+
+def is_llm_expression(expr: AbstractExpression):
+    if isinstance(expr, FunctionExpression) and expr.name.lower() in LLM_FUNCTIONS:
+        return True
+    return False
+
+
+def extract_llm_expressions_from_project(exprs: List[AbstractExpression]):
+    remaining_exprs = []
+    llm_exprs = []
+    for expr in exprs:
+        if is_llm_expression(expr):
+            llm_exprs.append(expr.copy())
```

Comment:
Add a note here, the chained function call will not work here. For example `STRTODATAFRAME(LLM('EXTRACT SOME COLUMN', data))`

Reply:
Yes, I'll add that in next PR